IliasDownloaderUniMA/__init__.py

```diff
@@ -4,8 +4,8 @@
 from bs4 import BeautifulSoup
 from urllib.parse import urljoin
 from pathlib import Path as plPath
-from dateutil.parser import parse as parsedate
-from datetime import datetime
+from dateutil.parser import parse 
```

Comment:
Should be

```suggestion
from dateutil.parser import parse as parsedate
```

(as before).

Reply:
Sorry! I don’t why know why it got removed.. it was not on purpose